vpc-endpoints.tf

```diff
@@ -12,7 +12,12 @@ resource "aws_vpc_endpoint" "s3" {
 
   vpc_id       = local.vpc_id
   service_name = data.aws_vpc_endpoint_service.s3[0].service_name
-  tags         = local.vpce_tags
+  tags = merge(
+    local.vpce_tags,
+    {
+      "Name" = "${var.name}-${var.vpc_endpoint_suffix}"
```

Comment:
Why do you want to override `Name` tag like this?

You can already set `var.vpc_endpoint_tags = { Name = "my-vpcname-vpc-endpoint" }` to have the same effect, no?

Reply:
true, but this to align with other resources the add the name tag. default for this resources is that there is no name in the console.

i'll close this for now while i think of a better solution. ill open a different PR for the doc fixes